README.md

```diff
@@ -1,19 +1,34 @@
-# About
+# About #
+**svneverever** helps you inspect the structure of a SVN repository and the changes made to it over time. Its most common use is in combination with [svn-all-fast-export](https://github.com/svn-all-fast-export/svn2git) (or "KDE's svn2git" if you wish).
 
-**svneverever** helps you inspect the structure of a SVN repository
-and the changes made to it over time.
-Its most common use probably is in combination with
-[svn-all-fast-export](https://github.com/svn-all-fast-export/svn2git)
-(or "KDE's svn2git" if you wish).
+# Installation #
 
+## Dependencies ##
+**svneverever** requires python 3.5 or higher and [pysvn](https://pysvn.sourceforge.io/). If you want to install **svneverever** with **pip** you need a few additional packages. For Debian/Ubuntu the following should work
 
-# Example
+```bash
+sudo apt install python3-setuptools python3-pip python3-svn
+```
 
-In the following shell session I am first dumping the SVN history of
-[headphone effect library bs2b](http://bs2b.sourceforge.net/)
-using [rsvndump](http://rsvndump.sourceforge.net/).
-(I could pass `svn://svn.code.sf.net/p/bs2b/code/` to **svneverever** directly
-but it would mean to download all history for each run to **svneverever**.)
+## Package manager ##
+Gentoo and Funtoo offer [a package](https://repology.org/project/svneverever/versions) for **svneverever**.
```

Comment:
While not your duty at all, would you be up for adding Arch AUR and openSUSE to this mix? (https://repology.org/project/python:svneverever/versions)